Sources/apollo-ios-pagination/PaginationInfo.swift

```diff
@@ -0,0 +1,3 @@
+public protocol PaginationInfo: Sendable {
```

Comment:
Have we actually removed the requirement that `PaginationInfo` conform to `Hashable`? I assume no since we are storing them as `AnyHashable` in the watcher above?

I'm not a huge fan of hidden requirements that don't fail at compile time, but will fail at runtime instead...

Reply:
> Have we actually removed the requirement that PaginationInfo conform to Hashable? I assume no since we are storing them as AnyHashable in the watcher above?

Actually, we've successfully removed it! We're storying the `Query.Variables` as `AnyHashable` above, not the pages themselves. There is a mapping from `Query.Variables` to the `Query.Data`, and that mapping is used to derive the current `PaginationInfo` we are operating over.